nixos/modules/services/misc/paperless.nix

```diff
@@ -331,6 +331,8 @@ in
         ExecStart = "${cfg.package}/bin/celery --app paperless beat --loglevel INFO";
         Restart = "on-failure";
         LoadCredential = lib.optionalString (cfg.passwordFile != null) "PAPERLESS_ADMIN_PASSWORD:${cfg.passwordFile}";
+        # Needs to talk to redis server for scheduling
+        PrivateNetwork = false;
```

Comment:
By default we use unix sockets, so this is not required. We should probably add a option to skip the local setup and let people freely configure whatever redis.

Reply:
That sounds reasonable @SuperSandro2000.